### ISSUE TITLE
This add-on breaks Anki - fields become invisible

### ISSUE THREAD

**Issue by sugarycat** [hgiesel/closet#106]
**Describe the bug**

1. When adding new cards with the Closet Anki add-on enabled, all of the fields of the note become invisible

<img width="653" alt="image" src="https://user-images.githubusercontent.com/109921798/185516510-3577d14c-4918-46a0-9f30-1228c7cae5ed.png">

2. Similarly, all of the fields of the note are invisible in the browser mode and tags are missing as well.
<img width="735" alt="image" src="https://user-images.githubusercontent.com/109921798/185516608-47bea0a1-ac91-4ea8-8bbd-717b9e6cb02f.png">

- Anki client: Windows 11, AnkiDesktop, Version 2.1.53 Qt6

All add-ons except Anki Closet were disabled

**Comment by Gr4ntaire** [comment]
I'm having this issue as well

**Comment by hgiesel** [comment]
I think it's called by an import call in javascript. Maybe you need to change something on your firewall, or security settings?

**Comment by Gr4ntaire** [comment]
I don't believe so, but I added an explicit exception for Anki and still see the issue.

**Comment by ccchan234** [comment]
> I think it's caused by an import call in javascript. Maybe you need to change something on your firewall, or security settings?

you addon need internet activity?

can we use it when no internet? thanks

**Comment by sugarycat** [comment]
Has there been any updates on this issue?
